tests/taskmodules/test_re_text_classification_with_indices.py

```diff
@@ -1199,34 +1197,35 @@ def test_encode_input_multiple_relations_for_same_arguments(
             # nor as skipped in statistics.
             assert candidate_relation_tuples == []
             if collect_statistics:
-                assert (
-                    caplog.messages[0] == "statistics:\n"
-                    "|                        |   per:founded_by |   per:founder |   all_relations |\n"
-                    "|:-----------------------|-----------------:|--------------:|----------------:|\n"
-                    "| available              |                1 |             1 |               2 |\n"
-                    "| skipped_same_arguments |                1 |             1 |               2 |"
-                )
+                assert statistics == {
+                    ("available", "per:founded_by"): 1,
+                    ("available", "per:founder"): 1,
+                    ("skipped_same_arguments", "per:founder"): 1,
+                    ("skipped_same_arguments", "per:founded_by"): 1,
+                }
+                assert caplog.messages == []
```

Comment:
why does the `expected_warning` only appear if `collect_statistics`? That should be independent of that. Is this a bug? Also in the other checks with `assert caplog.messages == []` below.